src/summary/LongitudinalTable.scss

```diff
@@ -8,24 +8,23 @@
 .table-scrollable {
     overflow-x: scroll !important;
 }
-.name {
-    &:hover {
-        color: #028DEA !important;
-        cursor: pointer !important;
-        background-color: $feature-gray !important;
-    }
-}
 .star-clicked {
     color: #028DEA !important;
 }
 .star-hovered {
-    color: #b3b3b3 !important;
+    color: #028DEA !important;
 }
 .star-cell {
     padding-left: 0px !important;
     padding-right: 0px !important;
     text-align: center !important;
     min-width: 30px !important;
+    &.star-body {
+        &:hover {
+            cursor: pointer !important;
+            background-color: $feature-gray !important;
```

Comment:
This might be picky but my personal thoughts is that the hover looks better without the grey background in the entire star cell.  Not sure if there are mockups for this or what other's opinions are but I would remove this background color on hover

Reply:
I talked to matt about this and it becomes hard to see which cell the mouse is hovered on when your cursor is between cells that are both selected like so
<img width="599" alt="Screen Shot 2019-06-25 at 1 47 33 PM" src="https://user-images.githubusercontent.com/46202699/60120925-f5967680-974f-11e9-8c0f-46a350e706f5.png">

Reply:
Can we do it so the hover background only shows up if it's starred or find another way to differentiate?